tooling/eslint-plugin/lib/config/jsx-a11y.ts

```diff
@@ -0,0 +1,86 @@
+import type { Linter } from "eslint";
+import { sourceFiles } from "../utils/patterns";
+
+const config: Linter.Config = {
+    overrides: [
+        {
+            files: sourceFiles,
```

Comment:
As there is only a single plugin and I doubt there would be others, I don,t think it's necessary to extract the files pattern in another file.

(As a side note, I am not a fan of this pattern, even in the web-configs project as it always force to be switch between 2 files to understand what files the sections is applying the rules for)

Reply:
Personally, this type of reusability enters the category of unnecessary coupling that could be replaced by copy/paste. I won't make a big deal of it thought for the web-configs project. Neither for this project, but it seems even more unnecessary here since it won't even be reused.

To answer your question, the only way I can think of that could make it easier for me would be if the name of the variables would include every file type that it covers.